packages/kernel-6.12/2001-Resolve-static-const-compiler-warnings.patch

```diff
@@ -0,0 +1,151 @@
+From 5d1dae633d16144e6fe21da2e70b179a0c009da3 Mon Sep 17 00:00:00 2001
+From: Martin Harriman <larvacea@mac.com>
+Date: Wed, 5 Mar 2025 20:37:48 +0000
+Subject: [PATCH] Resolve static const compiler warnings
```

Comment:
Why is this needed for 6.12 but not for the other kernels? We use the same GCC and it should be the same Neuron sources.

Reply:
It's probably because I haven't updated the Neuron SRPM. I will fix that and see.

Reply:
This patch doesn't appear to be applied by the spec file, so it seems like it can be dropped.